en/docs/deploy-and-publish/publish-on-dev-portal/publish-aws-apis-in-the-dev-portal.md

```diff
@@ -0,0 +1,118 @@
+# Importing APIs from AWS API Gateway to the WSO2 API Manager  
+
+This feature helps publish an API a user has in the AWS API gateway, on the WSO2 developer portal. These APIs will not be deployed into the WSO2 API gateway and will not be managed by the WSO2 API gateway.
+
+## Prerequisites 
+
+1. Two WSO2 API Controller commands are needed for this task. Hence the user needs to have the APICTL downloaded and configured. The apictl is a command line interface similar to the AWS CLI. [Getting started with the API Controller](https://apim.docs.wso2.com/en/latest/learn/api-controller/getting-started-with-wso2-api-controller/)
```

Comment:
Shall we change this as follows?
```suggestion
1. Two WSO2 API Controller commands are needed for this task. Hence the user needs to have the APICTL downloaded and configured. The apictl is a command-line interface similar to the AWS CLI. [Getting started with the API Controller](https://apim.docs.wso2.com/en/latest/learn/api-controller/getting-started-with-wso2-api-controller/)
```

Reply:
As mentioned previously, lets drop the sentence `The apictl is a command line interface similar to the AWS CLI`